backend/routes/usernameRoute.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const router = express.Router();
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+
+
+router.post('/check-username',  async (req, res) => {
+    const {username} = req.body;
+    const findUser = await prisma.user.findUnique({
+        where: {username}
+    })
+
+    const exists = findUser !== null
+    res.json({exists});
```

Comment:
We probably want to send back to the frontend something more descriptive. Usually we send status codes and messages back so we can use them to throw errors or success messages in the frontend.

look something along the lines of this: res.status(...).json({message: ...})

Reply:
do you want me to fix this or should i just get rid of this endpoint?

Reply:
fixed